docs/src/sphinxext/generate_package_rst.py

```diff
@@ -13,7 +13,7 @@
 # list of tuples for modules to exclude.  Useful if the documentation throws
 # warnings, especially for experimental modules.
 exclude_modules = [
-    ("experimental/raster", "iris.experimental.raster")  # gdal conflicts
+    ("experimental/raster", "iris.experimental.raster"),  # gdal conflicts
```

Comment:
this isn't on master.  Is it explicitly added for UGRID work?

Reply:
Looks like the extra comma is an artifact of this list being expanded in #4080, then the additional tuple being removed again in #4114.

Reply:
you can keep it as long as the change isn't in the 3.0.x branch. Personally if not needed i'd remove it

Reply:
I can see that side. I'm mostly put off by the prospect of a noisy commit history since we're not using the squash strategy for this PR. Is that a non-issue for you - an extra commit to remove the comma is fine?